cmd/opm/alpha/template/composite.go

```diff
@@ -73,5 +180,7 @@ and a 'composite template' file`,
 	cmd.Flags().BoolVar(&validate, "validate", true, "whether or not the created FBC should be validated (i.e 'opm validate')")
 	cmd.Flags().StringVarP(&compositeFile, "composite-config", "c", "composite.yaml", "File to use as the composite configuration file")
 	cmd.Flags().StringVarP(&catalogFile, "catalog-config", "f", "catalogs.yaml", "File to use as the catalog configuration file")
+	cmd.Flags().StringVar(&dockerfileDir, "dockerfile", "", "Generates a Dockerfile in the provided directory")
+	cmd.Flags().StringVar(&devfileDir, "devfile", "", "Generates a Devfile in the provided directory")
```

Comment:
I could be wrong here, but I thought we already know what directories we would need to output these files to? If so, I think these should be boolean flags to signal whether or not to include the creation of them in the correct output directories. IIRC the structure of the output with both devfiles and a dockerfile should look something like:
```sh
catalogs
├── foo
│   ├── catalog
│   │   └── README.md
│   └── devfile.yaml
├── bar
│   ├── catalog
│   │   └── README.md
│   └── devfile.yaml
└── catalog.Dockerfile
```
With this expected directory structure, I *think* we should be able to parse all the correct directory information from the various config files